pkg/acsengine/params_k8s.go

```diff
@@ -24,32 +24,32 @@ func assignKubernetesParameters(properties *api.Properties, parametersMap params
 		k8sVersion := orchestratorProfile.OrchestratorVersion
 		k8sComponents := api.K8sComponentsByVersionMap[k8sVersion]
 		kubernetesConfig := orchestratorProfile.KubernetesConfig
+		kubernetesImageBase := kubernetesConfig.KubernetesImageBase
 
 		if kubernetesConfig != nil {
-
 			if helpers.IsTrueBoolPointer(kubernetesConfig.UseCloudControllerManager) {
-				kubernetesCcmSpec := kubernetesConfig.KubernetesImageBase + k8sComponents["ccm"]
+				kubernetesCcmSpec := kubernetesImageBase + k8sComponents["ccm"]
 				if kubernetesConfig.CustomCcmImage != "" {
 					kubernetesCcmSpec = kubernetesConfig.CustomCcmImage
 				}
 
 				addValue(parametersMap, "kubernetesCcmImageSpec", kubernetesCcmSpec)
 			}
 
-			kubernetesHyperkubeSpec := kubernetesConfig.KubernetesImageBase + k8sComponents["hyperkube"]
+			kubernetesHyperkubeSpec := kubernetesImageBase + k8sComponents["hyperkube"]
 			if kubernetesConfig.CustomHyperkubeImage != "" {
 				kubernetesHyperkubeSpec = kubernetesConfig.CustomHyperkubeImage
 			}
 
 			addValue(parametersMap, "kubeDNSServiceIP", kubernetesConfig.DNSServiceIP)
 			addValue(parametersMap, "kubernetesHyperkubeSpec", kubernetesHyperkubeSpec)
-			addValue(parametersMap, "kubernetesAddonManagerSpec", cloudSpecConfig.KubernetesSpecConfig.KubernetesImageBase+k8sComponents["addonmanager"])
```

Comment:
`kubernetesImageBase` is derived from `cloudSpecConfig.KubernetesSpecConfig.KubernetesImageBase`. `defaults.go` does that. So I don't think it will break china contexts.

Reply:
In china contexts `cloudSpecConfig.KubernetesSpecConfig.KubernetesImageBase` will return the mooncake image base.

Reply:
Great, thanks for confirming!